doc/rst/language/spec/modules.rst

```diff
@@ -1207,9 +1207,12 @@ multiple modules.
 
    *Implementation Notes*.
 
-   In the current Chapel compiler implementation, the *– –main-module* flag
-   can be used to specify the module from which the main function
-   definition will be used.
+   A ``main`` function will only be considered to identify the main
+   module if it appears within a file named on the command line.
+
+   The *––main-module* flag can be used to specify the module from which
+   the main function definition will be used.
```

Comment:
I realize what you've got here for `--main-module` is similar to / no worse than what was here before, but I'd be inclined to say "to specify the module whose main() function will be used." and maybe "This is particularly useful in the event that multiple modules define a function named `main()`.

I also find myself (again) cringing at our use of `function` here rather than `procedure`, but I know that that's a piece of yarn you probably don't want to tug on—though I think we could change these cases without others.

Reply:
Ah that is #17769 again. I will change it to talk about the `main procedure` since that is consistent with both our current and probably future terminology.